src/main/java/com/example/document/client/EmployeeClient.java

```diff
@@ -19,6 +21,11 @@ public class EmployeeClient {
     public EmployeeDTO getEmployeeById(long employeeId) {
         String url = fromHttpUrl(employeeUrl + "/employee/{employeeId}")
                 .buildAndExpand(employeeId).toUriString();
-        return restTemplate.getForObject(url, EmployeeDTO.class);
+        try {
+            return restTemplate.getForObject(url, EmployeeDTO.class);
+        } catch (HttpClientErrorException exception) {
```

Comment:
We should catch not just any HttpClientErrorException, but exactly NOT_FOUND exception since we want to filter specific not_found case